src/js/chartAuthoring.js

```diff
@@ -123,6 +125,26 @@ https://raw.githubusercontent.com/fluid-project/chartAuthoring/master/LICENSE.tx
                                     "onPieChartReady.escalate": "{chartAuthoring}.events.onPieChartReady.fire"
                                 }
                             }
+                        },
+
+                        sonifier: {
+                            type: "floe.chartAuthoring.sonifier",
+                            createOnEvent: "{chartAuthoring}.events.onPieChartReady",
+                            options: {
+                                model: {
+                                    "dataSet": "{dataEntryPanel}.model.dataSet",
+                                    // We relay currentlyPlayingData and isPlaying to the overall component so that sonification play events can be used to change the rest of the interface
+                                    currentlyPlayingData: "{chartAuthoring}.model.currentlyPlayingData",
+                                    isPlaying: "{chartAuthoring}.model.isPlaying"
+
+                                },
+                                listeners: {
+                                    "onCreate.escalate": {
+                                        funcName: "{chartAuthoring}.events.onSonifierReady.fire",
+                                        priority: "last"
```

Comment:
I think this was a legacy from when the sonifier had a different createOnEvent. It's not an issue now that it waits until after the `onPieChartReady` event fires. Will remove priority.